src/Builder.hs

```diff
@@ -98,11 +98,14 @@ isOptional = \case
     Objdump  -> True
     _        -> False
 
--- TODO: Get rid of fromJust.
```

Comment:
By the way, I actually wanted something more in this TODO: I think eventually we'll be able to get rid of partiality here altogether, by keeping the path to the executable in the `Package` datatype, e.g. `data Package = Library | Package FilePath`. So, perhaps, let's change the TODO to `Get rid of partiality`?